### PR TITLE
Upgrade edxapp to hawthorn

### PR DESCRIPTION
**Pull request by jmaupetit**
## Purpose

Now that Open edX "hawthorn" has been released, let's see how we can deploy it with Arnold.

## Proposal

- [x] remove SQL dump load as it's no longer required with hawthorn (migrations are fast, like really, really fast)
- [x] switch to fun-plaftorm's hawthorn image
- [x] copy static files already generated by `update_assets` in the docker image to the statics PVC before collecting/post-processing statics (*)
- [x] upgrade to `hawthorn.1` base image

(*) for unclear reasons, `update_assets` generates at least a required file (_e.g._ `webpack-stats.json`) in the `STATIC_ROOT` directory whether `DEBUG` is true or false...

### CODE REVIEW

File: apps/edxapp/templates/mysql/job_00_load_sql_dump.yml.j2 [hunk 1]
```diff
@@ -1,42 +0,0 @@
-apiVersion: batch/v1
```
Comment (sampaccoud) [inline apps/edxapp/templates/mysql/job_00_load_sql_dump.yml.j2:1]: You dump this we still need it for ginkgo no?.... :scream_cat:

File: apps/edxapp/templates/cms/job_01_collectstatic.yml.j2 [hunk 2]
```diff
@@ -52,25 +52,28 @@ spec:
             - "-c"
             - cp -r /config/* /tmp/config &&
               cp -rL /tmp/configmap-cms/* /tmp/config/cms/ &&
-              cp /tmp/secret/credentials.vault.yml /tmp/config/cms/secrets.yml
+              cp /tmp/secret/credentials.vault.yml /tmp/config/cms/secrets.yml &&
+              cp -rL /edx/app/edxapp/staticfiles/* /tmp/staticfiles/
```
Comment (sampaccoud) [inline apps/edxapp/templates/cms/job_01_collectstatic.yml.j2]: We should add a long comment to make sure we remember the why and how of this hack + an issue to try to make a PR to edX and solve it at the root of the problem?

File: apps/edxapp/templates/mysql/job_00_load_sql_dump.yml.j2 (hunk 1 above)
Comment (jmaupetit) [reply apps/edxapp/templates/mysql/job_00_load_sql_dump.yml.j2:1]: Yup. We'll never push ginkgo to production, so there will be no need to keep track of this legacy trick. That said, I've already tagged the "latest release" that used it: https://github.com/openfun/arnold/releases/tag/ginkgo.1-1.0.6

File: apps/edxapp/templates/cms/job_01_collectstatic.yml.j2 (hunk 2 above)
Comment (jmaupetit) [reply apps/edxapp/templates/cms/job_01_collectstatic.yml.j2]: Working on it (the comment :wink:).

File: .circleci/config.yml [hunk 3]
```diff
@@ -222,7 +222,7 @@ jobs:
           name: Test the "edxapp" application bootstrapping (with redis enabled)
           command: |
             bin/ci-bootstrap "edxapp,redis"
-            bin/ci-test-service "cms" "Welcome to Open edX Studio"
+            bin/ci-test-service "cms" "Welcome to Your Platform Studio"
```
Comment (sampaccoud) [inline .circleci/config.yml]: lol

File: apps/edxapp/vars/all/main.yml [hunk 4]
```diff
@@ -6,7 +6,7 @@ edxapp_lms_host: "lms.{{ project_name}}.{{ domain_name }}"
 
 # -- edxapp (cms/lms)
 edxapp_image_name: "fundocker/edxapp"
-edxapp_image_tag: "ginkgo.1-1.0.6"
+edxapp_image_tag: "hawthorn.1rc3-1.0.2"
```
Comment (sampaccoud) [inline apps/edxapp/vars/all/main.yml]: hawthorn.1-1.0.0 is now available :taco: 
here: https://hub.docker.com/r/fundocker/edxapp/tags/

Comment (jmaupetit) [reply apps/edxapp/vars/all/main.yml]: That's on the roadmap of this PR!

File: apps/edxapp/templates/cms/job_01_collectstatic.yml.j2 [hunk 5]
```diff
@@ -52,25 +52,29 @@ spec:
             - "-c"
             - cp -r /config/* /tmp/config &&
               cp -rL /tmp/configmap-cms/* /tmp/config/cms/ &&
-              cp /tmp/secret/credentials.vault.yml /tmp/config/cms/secrets.yml
+              cp /tmp/secret/credentials.vault.yml /tmp/config/cms/secrets.yml &&
+              mkdir -p /tmp/staticfiles/studio &&
+              cp /edx/app/edxapp/staticfiles/studio/webpack-stats.json /tmp/staticfiles/studio/
```
Comment (sampaccoud) [inline apps/edxapp/templates/cms/job_01_collectstatic.yml.j2:65]: add comment as discussed?

File: apps/edxapp/templates/lms/job_02_collectstatic.yml.j2 [hunk 6]
```diff
@@ -52,25 +52,28 @@ spec:
             - "-c"
             - cp -r /config/* /tmp/config &&
               cp -rL /tmp/configmap-lms/* /tmp/config/lms/ &&
-              cp /tmp/secret/credentials.vault.yml /tmp/config/lms/secrets.yml
+              cp /tmp/secret/credentials.vault.yml /tmp/config/lms/secrets.yml &&
+              cp /edx/app/edxapp/staticfiles/webpack-stats.json /tmp/staticfiles/
```
Comment (sampaccoud) [inline apps/edxapp/templates/lms/job_02_collectstatic.yml.j2:64]: add comment also here?

**Review by sampaccoud — APPROVED** [review]

File: apps/edxapp/templates/cms/job_01_collectstatic.yml.j2 (hunk 5 above)
Comment (jmaupetit) [reply apps/edxapp/templates/cms/job_01_collectstatic.yml.j2:65]: Done.

File: apps/edxapp/templates/lms/job_02_collectstatic.yml.j2 (hunk 6 above)
Comment (jmaupetit) [reply apps/edxapp/templates/lms/job_02_collectstatic.yml.j2:64]: Done

**Comment by sampaccoud** [comment]
:ship: it!

**Pull request merged**